File: Java/JisooChoi/java/test/src/homework/fourth/board/Board.java
package homework.fourth.board;

public class Board {
    //필드
    private int boardNum;
    private String title;
    private String writer;
    private String content;

    //생성자
    public Board(){

    }

    public Board(int boardNum, String title, String writer, String content){
        this.boardNum = boardNum;
        this.title = title;
        this.writer = writer;
        this.content = content;
    }

    //메서드
    public void showInfo(){
        System.out.println("------ 게시판 작성 ------");
        System.out.println("게시판 "+boardNum+"번");
        System.out.println("제목: "+title);
        System.out.println("작성자: "+writer);
        System.out.println("내용: "+content);
    }
}

File: mfe_react_vue/JinHoKim/mfe/react-board-app/src/api/board.js
import axios from "axios"

export const fetchBoardList = () => axios.get("http://localhost:7777/jpa-board/list")

export const registerBoard = (title, content, writer) => 
    axios.post("http://localhost:7777/jpa-board/register", { title, content, writer })

export const fetchBoard = (boardId) => axios.get(`http://localhost:7777/jpa-board/${boardId}`)

export const removeBoard = (boardId) => axios.delete(`http://localhost:7777/jpa-board/${boardId}`)

export const modifyBoard = (boardId, title, content, writer) => 
    axios.put(`http://localhost:7777/jpa-board/${boardId}`, { title, content, writer })